mobile/components/screens/Register.js

```diff
@@ -3,25 +3,60 @@ import { View, Text, TextInput, TouchableOpacity } from 'react-native';
 import { SafeAreaView } from 'react-native-safe-area-context';
 import Alert from '../utils/Alert';
 import LongLogo from "../../assets/longlogo.svg";
-import { validateRegisterInputs } from '../utils/Validator';
+import DropDownPicker from 'react-native-dropdown-picker'; // Import the dropdown picker
 
 const RegisterScreen = ({ navigation }) => {
   const [formData, setFormData] = useState({
+    email: '',
     firstName: '',
     lastName: '',
-    email: '',
+    gender: '',
     password: '',
     confirmPassword: '',
   });
-
   const [alertMessage, setAlertMessage] = useState(null);
 
-  const handleRegister = () => {
-    if (validateRegisterInputs(formData, setAlertMessage)) {
-      console.log("Rejestracja udana:", formData);
+  const [open, setOpen] = useState(false);
+  const [gender, setGender] = useState(formData.gender);
+
+  const handleRegister = async () => {
+    if (formData.password !== formData.confirmPassword) {
+      setAlertMessage('Hasła się nie zgadzają.');
+      console.log('Hasła się nie zgadzają:', formData.password, formData.confirmPassword);
+      return;
     }
-  };
 
+    console.log('Wysyłane dane:', formData);
+
+    try {
+      const response = await fetch('http://10.0.2.2:5253/api/account/register', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          firstName: formData.firstName,
+          lastName: formData.lastName,
+          email: formData.email,
+          password: formData.password,
+          gender: gender,
+        }), 
+      });
+
+      console.log('Odpowiedź serwera:', response);
+
+      if (response.ok) {
+        setAlertMessage('Rejestracja zakończona sukcesem. Możesz się teraz zalogować.');
+        // navigation.navigate('Login');
```

Comment:
to bym odkomentował bo spoko flow jest z nawigacją

Reply:
to muszę alerta wywalić na login screena bo znika